understanding/non-text-content.html

```diff
@@ -1730,62 +1458,59 @@ <h2>Key Terms</h2>
                   <dd>
                      <definition xmlns="">
                         
                         
-                        <p xmlns="http://www.w3.org/1999/xhtml">sequence of characters that can be <a href="#dfn-programmatically-determined">programmatically determined</a>, where the sequence is expressing something in <a href="#dfn-human-language">human language</a>
+                        <p xmlns="http://www.w3.org/1999/xhtml"><a href="#dfn-programmatically-determined">プログラムによる解釈</a>が可能な文字の並びで、<a href="#dfn-human-language">自然言語</a>で何かを表現しているもの。
                            
                         </p>
                         
                         
                      </definition>
                   </dd>
-                  <dt id="dfn-text-alternative">text alternative</dt>
+                  <dt id="dfn-text-alternative">テキストによる代替 (text alternative)</dt>
                   <dd>
                      <definition xmlns="">
                         
                         
                         <p xmlns="http://www.w3.org/1999/xhtml">
-                           <a href="#dfn-text">Text</a> that is programmatically associated with <a href="#dfn-non-text-content">non-text content</a> or referred to from text that is programmatically associated with non-text content.
-                           Programmatically associated text is text whose location can be programmatically determined
-                           from the non-text content.
+                           <a href="#dfn-text">非テキストコンテンツ</a>とプログラムで関連付けられる<a href="#dfn-non-text-content">テキスト</a>、又は非テキストコンテンツとプログラムで関連付けられるテキストから参照されるテキスト。プログラムで関連付けられたテキストとは、その場所を、非テキストコンテンツからプログラムによる解釈が可能なテキストである。
                            
                         </p>
                         
                         
                         <aside xmlns="http://www.w3.org/1999/xhtml" class="example">
                            
-                           <p>An image of a chart is described in text in the paragraph after the chart. The short
-                              text alternative for the chart indicates that a description follows.
+                           <p>チャートの画像があり、その直後の段落にテキストによる説明がある。チャートに対する短いテキストによる代替で後に説明があることを示している。
                               
                            </p>
                            
                         </aside>
                         
                         
                         <div xmlns="http://www.w3.org/1999/xhtml" class="note">
-                           <p class="note-title marker">Note</p>
-                           <p>Refer to <a href="https://www.w3.org/WAI/WCAG21/Understanding/conformance#text-alternatives">Understanding Text Alternatives</a> for more information.
+                           <p class="note-title marker">注記</p>
+                           <p>より詳細な情報は、<a href="https://www.w3.org/WAI/WCAG21/Understanding/conformance#text-alternatives">テキストによる代替を理解する</a> を参照。
```

Comment:
ここは原文がミスってますが、とりあえずWAICのURLにしれっと差し替えちゃいましょうか…？
```suggestion
                           <p>より詳細な情報は、<a href="https://waic.jp/translations/WCAG22/Understanding/conformance#text-alternatives">テキストによる代替を理解する</a> を参照。
```

Reply:
話した結果、原文URLままとする方向で。